pkgs/development/interpreters/python/hooks/pypa-build-hook.sh

```diff
@@ -0,0 +1,19 @@
+# Setup hook to use for pypa/build projects
+echo "Sourcing pypa-build-hook"
+
+pypaBuildPhase() {
+    echo "Executing pypaBuildPhase"
+    runHook preBuild
+
+    echo "Creating a wheel..."
+    @pythonInterpreter@ -m build --no-isolation --outdir dist/ $pypaBuildFlags
```

Comment:
```suggestion
    @pythonInterpreter@ -m build --no-isolation --outdir dist/ "$pypaBuildFlags"
```
perhaps

Reply:
> perhaps

The `$pipInstallFlags` for instance are not quoted... That can be changed of course but we might as well be consistent?

Reply:
> Isn't `pypaBuildFlags` a space separated list of arguments for `-m build`, that would collapse into a single argument if it were quoted?

That is correct - just tested that.